assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -184,4 +216,12 @@
 		display: flex;
 		justify-content: flex-end;
 	}
+
+	.googlesitekit-header__children > div:not(:first-child) {
+		margin-left: $grid-gap-phone;
+
+		@media (min-width: $bp-desktop) {
+			margin-left: $grid-gap-desktop; // There is a large margin between the search icon and the date range
```

Comment:
@hussain-t I'am not sure the comment is still applicable. Can you remove it if it's the case? 😁